### DISCUSSION TITLE
How to Apply a Specific Style to Parent Element when it Doesn't Have a Child with a Specific Class

### DISCUSSION THREAD

**Discussion by MonaAghili** [tailwindlabs/tailwindcss#14090] [Help] [answered]
In this example, I want to check if the parent doesn't have a child with `divider-content`. Currently, it adds a border if a child with `divider-content` exists, but I want to check if it doesn't exist to add this style instead.
And here’s the updated code snippet to reflect the desired logic:

```
<div
  class="has-[.divider-content]:border-t"
>
  <div class="divider-content"></div>
</div>
```

**Comment by wongjn** [comment, marked as answer]
Use `:not()`: https://play.tailwindcss.com/xbbYoH9h6w

**Reply by MonaAghili** [reply]
Thank you
